ubl/Handlers/Common/AddressHandler.php

```diff
@@ -43,8 +43,8 @@ public function return_supplier_party( $data, $options = array() ) {
 	}
 
 	public function return_supplier_party_details() {
-		$company    = ! empty( $this->document->order_document ) ? $this->document->order_document->get_shop_name()       : '';
-		$address    = ! empty( $this->document->order_document ) ? $this->document->order_document->get_shop_address()    : get_option( 'woocommerce_store_address' );
+		$company    = ! empty( $this->document->order_document ) ? wpo_ips_ubl_sanitize_string( $this->document->order_document->get_shop_name() ): '';
+		$address    = ! empty( $this->document->order_document ) ? wpo_ips_ubl_sanitize_string( $this->document->order_document->get_shop_address() ) : get_option( 'woocommerce_store_address' );
```

Comment:
I would apply the function solely to the output; otherwise, the fallback won't be handled properly.

By doing so, I would also apply the same approach to `$company` to maintain consistency.